README.md

```diff
@@ -83,10 +83,15 @@ snap connect gotop-cjbassi:system-observe
 - Process actions:
   - `<Tab>`: toggle process grouping
   - `dd`: kill selected process or group of processes
-- Process sorting
+- Process sorting:
   - `c`: CPU
   - `m`: Mem
   - `p`: PID
+- Process filtering:
+  - /: start editing filter
+	- (while editing):
+		- <Enter> accept filter
+		- <C-c>: clear filter
```

Comment:
These need to be backticked in order to be rendered properly.

Reply:
Done. Also converted the tabs to spaces.